recipe/build.sh

```diff
@@ -8,6 +8,7 @@ cmake .. \
     -DCMAKE_FIND_ROOT_PATH=$PREFIX \
     -DCMAKE_INSTALL_LIBDIR=lib \
     -DREGEX_BACKEND=pcre2 \
-    -DCMAKE_BUILD_TYPE=Release
+    -DCMAKE_BUILD_TYPE=Release \
+    -DUSE_SSH=ON
```

Comment:
Should we adjust CMake options for our build? See https://github.com/libgit2/libgit2/blob/fbea439d4b6fc91c6b619d01b85ab3b7746e4c19/.github/workflows/main.yml#L158

Reply:
Not sure. I would say if it ain't broke, don't fix it?